From: Armin Novak <armin.novak@thincast.com>
Date: Tue, 22 Mar 2022 09:51:08 +0100
Subject: Removed unnecessary assert

(cherry picked from commit 4b1427639a9e29f911d92c0f33175e191cd5a503)
(cherry picked from commit 34e851209a742a67604191759730bef3c332ebcd)
---
 winpr/libwinpr/sspi/NTLM/ntlm_compute.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/winpr/libwinpr/sspi/NTLM/ntlm_compute.c b/winpr/libwinpr/sspi/NTLM/ntlm_compute.c
index 5360a2c..202240b 100644
--- a/winpr/libwinpr/sspi/NTLM/ntlm_compute.c
+++ b/winpr/libwinpr/sspi/NTLM/ntlm_compute.c
@@ -656,7 +656,6 @@ void ntlm_generate_random_session_key(NTLM_CONTEXT* context)
 void ntlm_generate_exported_session_key(NTLM_CONTEXT* context)
 {
 	WINPR_ASSERT(context);
-	WINPR_ASSERT(sizeof(context->ExportedSessionKey) == sizeof(context->RandomSessionKey));
 
 	CopyMemory(context->ExportedSessionKey, context->RandomSessionKey,
 	           sizeof(context->ExportedSessionKey));
